docs/modules/ROOT/pages/vshn-managed/minio.adoc

```diff
@@ -0,0 +1,6 @@
+= MinIO by VSHN
+
+MinIO by VSHN is currently only available on APPUiO Managed on request.
```

Comment:
The NAV says "On-Premise" and here it says "APPUiO Managed" which isn't necessarily On-Premise.

Reply:
That's true, but "on request" implies that you anyway have to ask